src/auth/UserRequest.h

```diff
@@ -247,7 +247,7 @@ int authenticateAuthUserRequestIPCount(Auth::UserRequest::Pointer);
 
 /// \ingroup AuthAPI
 /// See Auth::UserRequest::authenticated()
-int authenticateUserAuthenticated(Auth::UserRequest::Pointer);
+bool authenticateUserAuthenticated(const Auth::UserRequest::Pointer);
```

Comment:
Please refer to previous discussions about unwanted `const` in declarations.

```suggestion
bool authenticateUserAuthenticated(Auth::UserRequest::Pointer);
```

Reply:
change reverted